### ISSUE TITLE
Password expiry popup incorrectly reports 7 day expiry

### ISSUE THREAD

**Issue by seisvelas** [FlowCrypt/flowcrypt-browser#2879]
Password expiry popup says 7 days regardless of how many days are set in the actual configuration (in my case, 3).

![FIRELIZARD](https://user-images.githubusercontent.com/44826516/89243595-f55ee080-d5c9-11ea-8c87-f545647402a7.jpg)

This was reported in a support email: https://mail.google.com/mail/u/5/#inbox/FMfcgxwJXCJQPCFVbRWHmvnPvbcGrLrW

**Comment by tomholub** [comment]
Thanks for triaging it. Someone will need to further evaluate if this is a frontend issue, or a backend issue (are the changed settings properly updated on backend, and also in local storage?).

Then it would be good to have a test for this, too. If we broke it once, we will eventually break it again unless we write a test.

**Comment by yantakus** [comment]
I've just tested it and it seems to work as expected. Initially, it says my password expires in 7 days. When I changed it to 3 in the settings and reloaded the window, it changed to 7 days:

![image](https://user-images.githubusercontent.com/2179726/97403642-dd1b0b00-1905-11eb-9d96-8dbcc09414f9.png)

**Comment by tomholub** [comment]
Alex later please have a look if you can still reproduce it, and/or how. Thanks!

**Comment by seisvelas** [comment]
>Alex later please have a look if you can still reproduce it, and/or how. Thanks!

It's working for me as well now. I'm not sure what we changed, to be honest, the only recent work that's been done on the relevant code as the refactor in https://github.com/FlowCrypt/flowcrypt-browser/pull/3012, which doesn't contain anything that should've made this work.

I tried to reproduce it in both Firefox and Chrome and it's working in both.
